Stratis.BitcoinD/Program.cs

```diff
@@ -23,30 +23,39 @@ public static void Main(string[] args)
 		{
 			Logs.Configure(new LoggerFactory().AddConsole(LogLevel.Trace, false));
 			NodeSettings nodeSettings = NodeSettings.FromArguments(args);
+			
+			try 
+			{
+				Checks.VerifyAccess(nodeSettings);
```

Comment:
You could actually only put this code in the try catch. 
And the try catch could go inside the method actually
But I will push it like this and you can fix in another PR if thats fine.